tests/api_tests/users/test_views.py

```diff
@@ -380,3 +381,215 @@ def test_get_404_path_nodes_user_id_no_user(self):
         url = "/{}nodes/{}/".format(API_BASE, self.user_one._id)
         res = self.app.get(url, expect_errors=True)
         assert_equal(res.status_code, 404)
+
+
+class TestUserUpdate(ApiTestCase):
+
+    def setUp(self):
+        super(TestUserUpdate, self).setUp()
+
+        self.user_one = AuthUserFactory.build()
+        self.user_one.fullname = 'Martin Luther King Jr.'
```

Comment:
This is fine, but for future reference, `build()` takes the same arguments as the model constructor, so you can do:

```python
AuthUserFactory.build(
    fullname='Martin Luther King Jr.',
    given_name = 'Martin',
    social=dict(
        github='userOneGithub',
        # ...
    ),
    # ...
)
```